### PR TITLE
Public Cloud: fix embargoed updates in transfer repos

### PR DESCRIPTION
**Pull request by jlausuch**
Parent: https://openqa.suse.de/tests/10434440
Child: https://openqa.suse.de/tests/10434442

### CODE REVIEW

File: tests/publiccloud/transfer_repos.pm [hunk 1]
```diff
@@ -37,7 +38,15 @@ sub run {
         # Those two variables contain list of repositories separated by comma
         set_var('MAINT_TEST_REPO', get_var('INCIDENT_REPO')) unless get_var('MAINT_TEST_REPO');
 
-        my @repos = split(/,/, get_var('MAINT_TEST_REPO'));
+        # We need to exclude embargoed incidents
+        my @all_repos = split(/,/, get_var('MAINT_TEST_REPO'));
+        my @repos;
+        my $incident;
+        for my $maintrepo (@all_repos) {
+            $incident = $1 while $maintrepo =~ /\/Maintenance:\/(\d+)/g;
```
Comment (pdostal) [inline tests/publiccloud/transfer_repos.pm:46]:
```suggestion
        for my $maintrepo (@all_repos) {
            my $incident = $1 while $maintrepo =~ /\/Maintenance:\/(\d+)/g;
```
No need for `$incident` outside of that condition.

**Review by pdostal — APPROVED** [review]
Cool! 🚀

**Review by asmorodskyi — APPROVED** [review]
LGTM

File: tests/publiccloud/transfer_repos.pm (hunk 1 above)
Comment (jlausuch) [reply tests/publiccloud/transfer_repos.pm:46]: Finally I left it as it was, cause perl doesn't like creating new variables inside loops :D

**Pull request merged**